ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -297,6 +298,24 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         self.leadingView = leadingView
     }
 
+    /// - Parameters:
+    ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
+    ///   - title: The title string.
+    ///   - backgroundColor: The background color of the `backgroundView`
+    @objc open func setup(style: Style, title: String, backgroundColor: UIColor) {
+        setup(style: style, title: title, accessoryButtonTitle: "")
+        customBackgroundColor = backgroundColor
+    }
+
+    /// - Parameters:
+    ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
+    ///   - attributedTitle: Title as an NSAttributedString for additional attributes.
+    ///   - backgroundColor: The background color of the `backgroundView`
+    @objc open func setup(style: Style, attributedTitle: NSAttributedString, backgroundColor: UIColor) {
```

Comment:
When it comes to using custom colors, the pattern we typically use in fluent is to add a public UIColor variable in the control instead of injecting the custom color in the setup functions (DrawerController, CardView for reference).

Reply:
Got it. I can also see that in **0.17.0** the `backgroundView` background color was changed to use `TableViewHeaderFooterViewTokenSet` values istead of `ColorToken`s. We need to use custom colors for some of our header views but didn't realize we were still using **0.14.1**.

Given that we can override the colors from our side by _registering_ a custom set of `ControlTokenValue`, it seems the change is no longer needed, we can just upgrade to > 0.17.0 and do that.

I'll go ahead and close this PR, sorry for the confusion.